cinema_service/settings.py

```diff
@@ -20,14 +26,12 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
The `ALLOWED_HOSTS` setting is currently set to `['*']`, which allows all hosts. This is not secure for a production environment. Consider specifying the exact hostnames or IP addresses that are allowed to access the application.